core/src/main/java/com/linecorp/armeria/common/ResponseEntityUtil.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2024 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.common;
+
+import com.linecorp.armeria.server.ServiceRequestContext;
+
+/**
+ * Utilities for {@link ResponseEntity}.
+ */
+public final class ResponseEntityUtil {
```

Comment:
If this is intended for an internal use, how about making it package-private or moving to `com.linecorp.armeria.internal.common`?